xsd/NeTEx_publication_timetable.xsd

```diff
@@ -1646,23 +1608,14 @@ Provides a general purose wrapper for NeTEx data content.</xsd:documentation>
 			<xsd:field xpath="@version"/>
 		</xsd:key>
 		<!-- =====EQUIPMENT Constraints========================= -->
-		<!-- =====Equipment unique========================== -->
-		<xsd:unique name="Equipment_UniqueBy_Id_Version">
-			<xsd:annotation>
-				<xsd:documentation>Every [Equipment Id + Version] must be unique within document.</xsd:documentation>
-			</xsd:annotation>
-			<xsd:selector xpath=".//netex:Equipment"/>
-			<xsd:field xpath="@id"/>
-			<xsd:field xpath="@version"/>
-		</xsd:unique>
 		<!-- =====Equipment Key ========================== -->
 		<xsd:keyref name="Equipment_KeyRef" refer="netex:Equipment_AnyVersionedKey">
 			<xsd:selector xpath=".//netex:EquipmentRef"/>
 			<xsd:field xpath="@ref"/>
 			<xsd:field xpath="@version"/>
 		</xsd:keyref>
 		<xsd:key name="Equipment_AnyVersionedKey">
-			<xsd:selector xpath=".//netex:Equipment | .//netex:PlaceEquipment | .//netex:InstalledEquipment | .//netex:ActualVehicleEquipment | .//netex:TicketValidatorEquipment | .//netex:TicketingEquipment   | .//netex:SanitaryEquipment  | .//netex:PassengerSafetyEquipment | .//netex:RubbishDisposalEquipment    |.//netex:PassengerInformationEquipment   | .//netex:EntranceEquipment  | .//netex:LiftEquipment |  .//netex:EscalatorEquipment | .//netex:TravelatorEquipment | .//netex:StaircaseEquipment | .//netex:RampEquipment  | .//netex:RoughSurface  | .//netex:PlaceLighting|  .//netex:PlaceSign |  .//netex:PlaceSign | .//netex:HeadingSign  | .//netex:GeneralSign   |.//netex:WaitingRoomEquipment   | .//netex:ShelterEquipment  | .//netex:SeatingEquipment | .//netex:TrolleyStandEquipment      | .//netex:LuggageLockerEquipment  |  .//netex:CycleParkingEquipment    |.//netex:AccessVehicleEquipment    |.//netex:WheelchairVehicleEquipment"/>
```

Comment:
Is the removal of *LuggageLockerEquipment* intended and OK?

Reply:
Certainly, it is an abstract element.
./netex_part_1/part1_ifopt/netex_ifopt_equipmentWaiting_version.xsd:	<xsd:element name="LuggageLockerEquipment" abstract="true" substitutionGroup="SiteEquipment">

Reply:
@skinkie  closing the conversationthen?